plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/cum_sum.cpp

```diff
@@ -11,8 +11,8 @@ using LayerTestsDefinitions::CumSumLayerTest;
 namespace {
 
 const std::vector<InferenceEngine::Precision> netPrecisions = {
-    InferenceEngine::Precision::I32,
-    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::I32, InferenceEngine::Precision::FP32,
+    // InferenceEngine::Precision::FP16,
```

Comment:
I think moving the commented-out option to the beginning of the list will clean up the linting (to enable one-per-line).